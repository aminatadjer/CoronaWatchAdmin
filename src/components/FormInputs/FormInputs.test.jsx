import React from 'react';
import ReactDom from 'react-dom';
import FormInputs from './FormInputs';
import FieldGroup from './FormInputs';
import { render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it('renders without crashing', ()=>{
    const div = document.createElement('div');
    ReactDom.render(<FormInputs ncols={10}></FormInputs>, div);
});

it('renders without crashing', ()=>{
    const div = document.createElement('div');
    ReactDom.render(<FieldGroup ncols={10}></FieldGroup>, div);
});

// check props

/*it('renders CustomButton correctly', ()=>{
    const {getByTestId} = render(<FieldGroup ncols={10} label="This is a label"></FieldGroup>);
    expect(getByTestId('field')).toHaveTextContent('This is a label');
});*/

//snapshots

it ('matches snapshot', ()=>{
    const tree = renderer.create(<FieldGroup ncols = {10} label="This is a label"></FieldGroup>).toJSON();
    expect(tree).toMatchSnapshot();
})

it ('matches snapshot', ()=>{
    const tree = renderer.create(<FieldGroup ncols={10}></FieldGroup>).toJSON();
    expect(tree).toMatchSnapshot();
})
