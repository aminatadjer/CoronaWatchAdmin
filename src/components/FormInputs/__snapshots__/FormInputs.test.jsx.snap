// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="row"
/>
`;

exports[`matches snapshot 2`] = `
<div
  className="row"
/>
`;
